lab-sarah/ng-template/app/component/searchbar/searchbar.html

```diff
@@ -0,0 +1,4 @@
+<div class="searchbar">
+<input ng-model="searchbarCtrl.searchTerm" type="text" placeholder="{{searchbarCtrl.searchAssets}}">
+<button ng-click="searchbarCtrl.clearSearch()">Clear</button>
```

Comment:
Good work implementing a clear button! You could have also written an expression that simply sets searchbarCtrl.searchTerm to null without using a function, but either way :)